### PR TITLE
updating install dependancies

### PR DESCRIPTION
**Pull request by nucleardreamer**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+enum34==1.1.6
+futures==3.2.0
+grpcio==1.14.1
+MySQL-python==1.2.5
+six==1.11.0
```
Comment (JeremyGrosser) [inline requirements.txt]: Do we really need to specify versions for six, futures, and enum34? I think grpcio will pull those in implicitly.

File: setup.py [hunk 2]
```diff
@@ -31,11 +31,14 @@
         ],
         packages=['kama'],
         install_requires=[
```
Comment (JeremyGrosser) [inline setup.py:33]: Seems awkward to have both install_requires in setup.py and requirements.txt. One or the other, not both.

Comment (0xVANISHED) [reply setup.py:33]: I didn't know which one should be preferred, I would assume just setup.py but it seemed appropriate to have something versioned for the tests as well hence the requirements.txt

npm has the concept of devDependancies but I did not see that for eggs at all. That would seem to be the goal right? Not bad to just stick mysql in the Dockerfile of course

Comment (JeremyGrosser) [reply setup.py:33]: You can add tests_require to setup.py

File: requirements.txt (hunk 1 above)
Comment (0xVANISHED) [reply requirements.txt]: no you're right, we don't need those explicitly

**Pull request merged**